chaoslib/litmus/pod_delete/pod-delete.j2

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-delete-{{ run_id }}
+  labels:
+    app: pod-delete
+    name: pod-delete-{{ run_id }}
+{% if chaos_uid is defined and chaos_uid != '' %}
+    chaosUID: {{ chaos_uid }}
+{% endif %}
+spec:
+  restartPolicy: Never
+  serviceAccountName: pod-delete-sa
```

Comment:
should it have custom support @ksatchit ?

Reply:
yes pls